frontend_app_html-document/src/container/PopupCreateHtmlDocument.jsx

```diff
@@ -96,10 +97,10 @@ class PopupCreateHtmlDocument extends React.Component {
       case 200:
         this.handleClose()
 
-        GLOBAL_dispatchEvent({ type: 'refreshContentList', data: {} })
+        GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.REFRESH_CONTENT_LIST, data: {} })
 
         GLOBAL_dispatchEvent({
-          type: 'openContentUrl', // handled by tracim_front:src/container/WorkspaceContent.jsx
+          type: CUSTOM_EVENT.OPEN_CONTENT_URL, // handled by tracim_front:src/container/WorkspaceContent.jsx
```

Comment:
now we have a variable, it is easier to find where are the usages of a given custom event.
So you can remove this kind of comments